src/web/demo/src/main/java/org/geoserver/web/demo/CommonFormatLink.java

```diff
@@ -0,0 +1,40 @@
+/* (c) 2019 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.web.demo;
+
+import org.apache.wicket.markup.html.link.ExternalLink;
+import org.geoserver.web.ToolLinkExternalInfo;
+
+public abstract class CommonFormatLink extends ToolLinkExternalInfo
```

Comment:
Add a javadoc for new public classes please, especially since this is an extension point that plugins can implement.

Reply:
OK